File: front/src/features/dashboard/data/modules.ts
import type { ModuleCardItem } from "../types";

export const moduleCards: ModuleCardItem[] = [
  {
    key: "dashboard",
    name: "Dashboard",
    status: "Planificado",
    description: "Resumen general del negocio, alertas y metricas clave.",
  },
  {
    key: "ventas",
    name: "Ventas",
    status: "Planificado",
    description: "Registro de ventas, detalle de items y medios de pago.",
  },
  {
    key: "caja",
    name: "Caja",
    status: "Planificado",
    description: "Aperturas, cierres, movimientos y control diario.",
  },
  {
    key: "cobros",
    name: "Cobros",
    status: "Planificado",
    description: "Seguimiento de pagos pendientes y cuenta corriente.",
  },
  {
    key: "inventario",
    name: "Inventario",
    status: "Planificado",
    description: "Stock, categorias, costos y actualizaciones de productos.",
  },
  {
    key: "clientes",
    name: "Clientes",
    status: "Planificado",
    description: "Ficha de clientes, historial y estados de cuenta.",
  },
];

File: back/app/api/routes/modules.py
from fastapi import APIRouter

from app.schemas.module import ModuleInfo

router = APIRouter()

MODULES = [
    ModuleInfo(key="dashboard", name="Dashboard", status="planned"),
    ModuleInfo(key="ventas", name="Ventas", status="planned"),
    ModuleInfo(key="caja", name="Caja", status="planned"),
    ModuleInfo(key="cobros", name="Cobros", status="planned"),
    ModuleInfo(key="inventario", name="Inventario", status="planned"),
    ModuleInfo(key="clientes", name="Clientes", status="planned"),
    ModuleInfo(key="categorias", name="Categorias", status="planned"),
    ModuleInfo(key="proveedores", name="Proveedores", status="planned"),
]


@router.get("/", response_model=list[ModuleInfo])
def list_modules() -> list[ModuleInfo]:
    return MODULES
